# VR-inserts

## pFC8, 9 and pFC53Tac transformation

pFC8, 9 and 53TacT1T2 are backbones for upcoming insert cloning experiments.
Here I am transforming E. coli to produce more of each plasmid.

### Protocol

Followed this [lab electroporation protocol](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/documents/Electroporation_of_E.coli.doc.pdf).

#### DNA

Measured DNA concentrations using nanodrop for each plasmid which are listed in the table below.

| Plasmid  | ng / ul |
| -------- | ------- |
| pFC8     | 922     |
| pFC9     | 1000    |
| pFC53Tac | 100     |

Image of plasmids used.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4755.jpg)

pFC8 and pFC9 were both diluted to a concentration of 100 mg / ul. 100 ng of DNA from each plasmid was used for the transformation.

During the first electroporation pFC8 arced and the bacteria were destroyed. I continued the protocol with the
pFC53Tac and pFC9 samples and plated each on ampicilin agarose gel and left to incubate overnight at 37 C. These plates became the "A" samples.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4756.jpg)

I then repeated the electro-poration protocol with three new samples (one of each plasmid) to both make up for the
fact the pFC8 bacteria were zapped and serve as replicates for the others in case something went wrong with the first batch, these became the "B" samples. When pulsing the B samples both pFC9 and pFC53Tac arced and so I was unable to do any replicates for those plasmids but thankfully pFC8 survived and so at least one of each plasmid transformation was plated. I plated the pulsed pFC8 cells onto two separate plates; rep 1 and 2 shown in the image below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4757.jpg)

| Series | Plasmid  | Replicate | Incubation start (pm) |
| ------ | -------- | --------- | --------------------- |
| A      | pFC9     | 1         | 12:40                 |
| A      | pFC53Tac | 1         | 12:40                 |
| B      | pFC8     | 1         | 1:10                  |
| B      | pFC8     | 2         | 1:10                  |

Tomorrow I will check on the colonies in the morning and
growing any survivors up over the next 24 hours.

## pFC8, 9 and pFC53Tac transformation day two

### Growing up bacteria

Picked up plates from incubation room at 8:00 am this morning (image of plates
below.)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4760.jpg)

All plates at usable colonies! I transfered one colony  pFC9 A, pFC53tac A and pFC8 B rep 1 placed into
~500 ml of LB broth with 0.5g of ampicilin and placed flasks on the shaker in the incubation room at 8:58 am.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4761.jpg)


However, this was a mistake that
I only realized until later in the day. With that much ampicilin the concentration
comes out to *1000* ug/ml instead of the target 100 ug/ml. So hopefully I just get bacteria that are 10x as strong.

In case everything dies I made up a new batch of 2L of LB broth and
autoclaved.

Additionally I looked through the MIDI-prep kit box and did not find
buffers P1 or P2 so I made both of those solutions following the
guides posted in the "kitchen".

Megan and I also found the location of the MCB common dishwashers which are located on the far wing of the first floor of the building with the DNA sculpture (LSB?) in the stairwell.

Tomorrow as long as bacteria are able to grow in the high amplicilin
broth will do MIDI prep of plasmids and restriction digest.
## pFC8, 9 and pFC53Tac transformation day three

### Colony overnight growth

Came into the lab around 8 am and found that the pFC8 and 53tac
samples grew but the pFC9 sample did not.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4763.jpg)

Plated two new pFC9 colonies into seperate 1L flasks with ~500 ml
LB and 0.05 g amplicilin (100ug/ml).

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4764.jpg)

Placed into incubation room at ~8:30 am to grow.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4765.jpg)

### pFC8 and 53tac midi-prep

Followed Qiagen midi-prep protocol. Used all Qiagen kit reagents for
pFC8 and all reagents I made on Tuesday for pFC53tac. Did not run restriction digest yet (still need to select enzymes to use). Results
from nanodrop are below.

#### pFC8 nanodrop results

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4769.jpg)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4770.jpg)

#### pFC53tac nanodrop results

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4768.jpg)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4767.jpg)

#### DNA concentrations

|  Plasmid  |    DNA (ng / ul) |
|  -------  |    -----------   |
|   pFC8    |      312         |
| pFC53tac  |      293         |

Both samples were stored in -20 C kitchen freezer in the `VR-inserts` box.

### Lessons learned

This was a hell of a day. A combination of not really knowing what
I was doing with this protocol and factors out of my control
(centrifuge compatibility hell) lead to this prep taking just around
11 hours which is absolutely insane. So I have put together list of
advice / tips for next midi-prep

- Cool all centrifuges to 4 C before starting anything
- Figure out what rotors you are using for each spin the day before
  - SLA 1500 works in MCB centrifuge room with the 250 ml centrifuge flasks. These will generally be your best bet for larger volumes.
  - The centrifuge that uses SLA 1500 takes about 45 mins to cool down to 4 C.
- The 50 ml centrifuge tubes with the weird caps *will* fit into the
lab's refrigerated centrifuge you just need to push down on the UFO
rotor a bit when closing it.
- Minimize the number of separately treated sample tubes before running the columns. Running columns with large volumes will be a
major time killer.
## pFC8, 9 and pFC53Tac transformation day three

### pFC9 midi prep

Collected pFC9 cultures at 8:30 am. Both had grown successfully. I kept the two samples pFC9-A-1 and pFC9-A-2 seperate during the rest of the procedure. I
followed the Qiagen midi-prep protocol using my own reagents for both samples.

![pFC9-A-2 cultures](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4771.jpg)

Added 10 ml stock RNAseA to 50 ml buffer P1 and chilled with buffer P3 before
starting protocol. 

For first spin to harvest bacteria I used lab refrigerated centrifuge at 6000 x g.
For spin after cell lysis I used MCB common centrifuge (floor 3) with 
SLA 1500 rotor from common centrifuge room on floor 2. Ran into problem where
centrifuge would not start but rebooting seemed to fix it. For final spins I
used refrigerated centrifuge in lab. All spins were at 4C. 

For sample pFC8-A-2 I used the Invitrogen maxi-prep column from 2007 to test
if these were still good and could be used in future preps.

![Invitrogen maxi prep column vs Qiagen midi prep column](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4772.jpg)

I followed the Qiagen maxi prep protocol for washing and eluting DNA from 
the Invitrogen column. 

### Results

#### pFC9-A-1 (Qiagen midi prep column)

Nanodrop results

![pFC9-A-1 nano drop results](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4774.jpg)

![pFC9-A-1 nanodrop curve](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4775.jpg)

#### pFC9-A-2 (Invitrogen maxi prep column)

Nanodrop results

![pFC9-A-2 nanodrop results](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4776.jpg)

![pFC9-A-2 nanodrop curve](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4778.jpg)

Basically no plasmid was captured my the Invitrogen column and these columns
should be thrown away or at least not used with Qiagen reagents. Absorbance curve showed a large degree of
contamination and this sample should not be used for anything. I saved it
in case it is needed for some reason but noted on the tube that it should
not be used due to contamination and low DNA concentration.

Both samples were stored in the `VR-inserts` box in the -20 C freezer on my shelf in the kitchen. 


## Confirming midi prep results

Today I am working on visualizing the results of the midi preps
I did for pFC8, 9 and 53tac (see notes from [7-20-21](1-7-20.md)
to [7-23-21](1-7-20.md)).

### KpnI + HindIII double digest

First I utilized a restriction digest with HindIII and KpnI
to confirm plasmid ids and check for contaminating DNAs. Link to
spreadsheet used for reagent calculation [can be found here](https://docs.google.com/spreadsheets/d/1TSMWX3cN_CJO4ygUxHQ6MJX4J-_2AXtZk6WVvlGa3e4/edit?usp=sharing).

#### Reagents

| Reagent    | Expiration | Lot number |
| ---------- | ---------- | ---------- |
| Buffer 2.1 | 5/23       | 10070034   |
| KpnI       | 11/19      | 0551711    |
| HindIII    | 8/22       | 10080749   |

KpnI was expired but it did not seem to significantly effect results.

#### Results

I expected to see results similar to the simulated gel shown below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC8-9-53-tac-HindIII-KpnI.png)

```
MW:  1 kb DNA Ladder

1:  pFC8
    HindIII + KpnI
       1. 2608 bp
       2. 981 bp

2:  pFC9
    HindIII + KpnI
       1. 2616 bp
       2. 973 bp

3:  pFC53tacT1T2
    HindIII + KpnI
       1. 2935 bp
       2. 1385 bb
```

I digested each plasmid for 30 mins at 37C and then ran them on
a 0.8% agarose TAE gel for 1 hr at 90 volts.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC8-9-53-tac-HindIII-KpnI-Digest.png)

Overall both pFC8 and pFC9 look as expect although they cannot
be distinguished with REs. pFC54tac does show the shift upwards
on the large fragment but in a less dramatic fashion as the simulated
gel. This is even more true for the smaller (expected 1385 bp) fragment which shows little differentiation from the ~980 bp fragments produced
from pFC8 and pFC9.

### BamHI-HF + KpnI-HF double digest

After running the KpnI HindIII digest and looking back at the plasmid
maps I realized there was a much better digest that I could have done
that would in theory resolve all three plasmids from each other. Digesting
with BamHI and KpnI takes advantage of the reversed position KpnI site on
pFC8 and 9 and using BamHI which is located downstream of the T1T2 terminators on pFC53tac (which pFC8 and 9 lack) means all three plasmids will produce unique fragments.

So I repeated by protocol but used BamHI-HF and KpnI-HF enzymes with CutSmart buffer. I used HF version because these cutters normally do not have good activity together.

#### Reagents


| Reagent         | Expiration | Lot number |
| --------------- | ---------- | ---------- |
| CutSmart Buffer | 9/22       | 10046090   |
| KpnI-HF         | 4/17       | 0061504    |
| BamHI-HF        | 8/17       | 0101508    |

#### Results

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC8-9-53-tac-BamHI-KpnI-Digest-labeled.png)

Here both pFC8 and pFC9 look like pFC8 in the simulated gel. This
makes me think that I accidentally added pFC8 to the pFC9 tube of
the transformation.

I think overall the results for pFC53tac conform to expectations, simulated gel for the two digests done below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC53tac_digests_7-26-21.png)

```
MW:  1 kb DNA Ladder

1:  pFC53tacT1T2
    KpnI + BamHI
       1. 2557 bp
       2. 1763 bp

2:  pFC53tacT1T2
    KpnI + HindIII
       1. 2935 bp
       2. 1385 bp
```

Tomorrow I will re-run this gel but using Fred's pFC8 and pFC9 (the same
plasmid used in notes from [7-20-21](1_7-20-21.md)) and start transformation
with pFC9 again because I am thinking that is the most likely explanation.
## Confirming identity of pFC9(?) midi prep plasmid

Today I am working on making sure pFC8 is 8 and pFC9 (or what I thought
was pFC9) is also pFC8.

### Double digest with BamHI and KpnI

Replicating restriction digest protocol from [yesterday](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/5_7-26-21.md)
to make sure that pFC9 is actually pFC8. Spreadsheet of reagents, volumes
and DNA concentrations is [available at this link](https://docs.google.com/spreadsheets/d/1TSMWX3cN_CJO4ygUxHQ6MJX4J-_2AXtZk6WVvlGa3e4/edit?usp=sharing).

#### Reagents

| Reagent         | Expiration | Lot number |
| --------------- | ---------- | ---------- |
| CutSmart Buffer | 9/22       | 10046090   |
| KpnI-HF         | 4/17       | 0061504    |
| BamHI-HF        | 8/17       | 0101508    |

#### Expected results

Simulated gel and lane contents are below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC8-9-53-BamHI-KpnI-digest.png)

```
MW:  1 kb DNA Ladder

1:  pFC8
       1. 3589 bp

2:  pFC8
    BamHI + KpnI
       1. 2558 bp
       2. 1031 bp

3:  pFC9
       1. 3589 bp

4:  pFC9
    KpnI + BamHI
       1. 3535 bp
       2. 54 bp

5:  pFC53tacT1T2
       1. 4320 bp

6:  pFC53tacT1T2
    BamHI + KpnI
       1. 2557 bp
       2. 1763 bp
```

### Results

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC8-9-midi-pFC8-9-stock-kpnI-bamhi-digest-labeled.png)

This looks like everything is pFC8!

Below is the tube I got "pFC9" from. I am pretty sure that says pFC9.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/IMG-4815.jpg)

Talked to Fred after and we looked at the original primers and constructs
used to make pFC8 and 9 and seems like the genbank file is incorrect. The
locations of the KpnI and EcoRI sites are not flipped. This would explain
the results of this digest. The pFC9 file has since been updated in the 
lab shared folder. I am currently reworking cloning strategy to reflect
this. Will likely require new sequences or at least new homology arms.

### Double digest with BamHI and NotI

The R-loop forming region in pFC8 and 9 is flipped and NotI, as a part of this region, is also flipped with it. Here I am digesting pFC8 and 9 with
BamHI (same location both plasmids) and NotI in order to tell them apart.

I followed exact same protocol as the BamHI + KpnI digest that is shown above but swapped KpnI-HF for NotI-HF. Should note that NotI-HF is very low and probably needs to be re-ordered.

#### Reagents

| Reagent         | Expiration | Lot number |
| --------------- | ---------- | ---------- |
| CutSmart Buffer | 9/22       | 10046090   |
| NotI-HF         | 12/20      | 10030791   |
| BamHI-HF        | 8/17       | 0101508    |

#### Expected results

Simulated digest is shown below. Only showing first four lanes as the next
four should be the exact same in theory.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pfc9-8-bamhi-notI.png)

```
MW:  1 kb DNA Ladder

1:  pFC8
       1. 3589 bp

2:  pFC8
    BamHI + NotI
       1. 3307 bp
       2. 282 bp

3:  pFC9
       1. 3589 bp

4:  pFC9
    NotI + BamHI
       1. 2801 bp
       2. 788 bp
```



#### Results

Ran gel 0.08 agarose for 1 hr at 120 volts with TAE and EtBr in running
buffer as well as in agarose.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/2021-07-27_15h42m32s_pFC8-9-midi-pFC8-9-stock-BamHI-NotI-digest-labeled.png)

Based on these results pFC9 is *not* misclassified as pFC8 and is in fact
correctly labeled. 

### Other misc. items

#### pFC9 cultures

Early in the day when I thought that my pFC9 midi prep sample was definitely
pFC8 I started another pFC9 transformation. Those bacteria are currently plated and incubating in at 37C. I placed them into the incubation room around 9:45 am after electroporation with pFC9. I used the eletro-competent E.coli in the first box accessible in the -80C freezer.

#### Amp agar plates

Made ~20 amplicilin agar petri plates for future transformations following the kitchen reagent sheet guidelines. I stored all plates in the plastic
sleeve they came in in the deli fridge.

### For tomorrow

Tomorrow need to checkout bacteria and want to confirm the identity and
orientation of the Tac promoter and T1T2 terminators using as many of
the digests below that we have enzymes for.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC53tac_digests.png)

```
MW:  1 kb DNA Ladder

1:  pFC53tacT1T2
       1. 4320 bp

2:  pFC53tacT1T2
    HindIII + KpnI
       1. 2935 bp
       2. 1385 bp

3:  pFC53tacT1T2
    EcoRI + XbaI
       1. 2574 bp
       2. 1746 bp

4:  pFC53tacT1T2
    BamHI + HindIII
       1. 3942 bp
       2. 378 bp

5:  pFC53tacT1T2
    SacI + BamHI
       1. 3258 bp
       2. 1062 bp

6:  pFC53tacT1T2
    SacI + KpnI
       1. 3619 bp
       2. 701 bp
```

Lane 1 establishes undigested plasmid, second is a double check of the
digest done on [7-26-21](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/5_7-26-21.md). Third digest is to partially confirm the presence of the T1T2 terminators since EcoRI is present in the
terminators. Fourth digest confirms the length of the T1T2 terminators since
this digest cuts that sequence out. Fifth and sixth digest together confirm
distances between T1T2 and tac promoter since both cutting at the sacI site which is at the midpoint of the two features.



## Confirming identity of pFC53tac

Today I am working on confirming the identity of the pFC53tac sample
that I prepared via midi prep using restriction digests.

### pFC53tac restriction digests

I digested ~200 ng of pFC53tac sample using the assay design below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC53-tac-midi-prep-digest.png)

```
MW:  1 kb DNA Ladder

1:  pFC53tacT1T2
       1. 4320 bp

2:  pFC53tacT1T2
    HindIII + KpnI
       1. 2935 bp
       2. 1385 bp

3:  pFC53tacT1T2
    EcoRI + XbaI
       1. 2574 bp
       2. 1746 bp

4:  pFC53tacT1T2
    MfeI + BamHI
       1. 3419 bp
       2. 901 bp

5:  pFC53tacT1T2
    MfeI + KpnI
       1. 3458 bp
       2. 862 bp

6:  pFC53tacT1T2
    MfeI + KpnI + BamHI
       1. 2557 bp
       2. 901 bp
       3. 862 bp
```

#### Reagents

Each reaction had a total volume of 10ul and used 10x CutSmart buffer. All
reactions were incubated at 37C for 30 mins.

| Reagent    | Lot number | Expiration date |
| ---------- | ---------- | --------------- |
| BamHI-HF   | 0101508    | 8/17            |
| KpnI-HF    | 0061508    | 8/17            |
| HindIII=HF | 0061508    | 8/17            |
| XbaI       | 0431803    | 3/20            |
| EcoRI-HF   | 10026002   | 10/20           |


#### Results

Ran 0.08 gel at 120 V for 45 mins in TAE. EtBr in agarose and running buffer.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC53tac-midi-prep-digest-labeled.png)

When first looking at these results the EcoRI-XbaI and digests using MfeI
did not make sense. I was not sure what was causing the double band in the
EcoRI-XbaI digest and it seemed like I forgot to add MfeI to those digests.
I talked to Fred about this and he mentioned that pFC8tac actually has
2 EcoRI sites which could explain the double band. pFC8tac also does not
have an MfeI site. I simulated this digest on pFC8tac and the results are
shown below.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_simulated_pFC8tac_digest.png)

This matches the results of the actual digest very well which leads me
to believe that the stock pFC53tac is mislabeled and is in fact pFC8tac. Tomorrow I will confirm this by repeating this same digest on the stock
I used in for the original midi prep and on pFC8tac stock.
## pFC53tac and pFC8tac digests plus primer design

Today I am continuing to work on determining the true identity and content
of the pFC53 plasmids. I also designed primers to amplify each plasmid for
sequencing.

### pFC53tac and pFC8tac digest

I digested Rachel's pFC53tac and pFC8tac and my own pFC53tac I prepared from
Rachel's pFC53tac stock. Below are the expected results for digests of true
pFC53tac and pFC8tac.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Gel_Image_pFC8tac_pFC53tac_EcoRI-Xba_BamHI-NotI_digest.png)

```
MW:  1 kb DNA Ladder

1:  pFC53tacT1T2
       1. 4320 bp

2:  pFC53tacT1T2
    BamHI + NotI
       1. 4299 bp
       2. 21 bp

3:  pFC53tacT1T2
    EcoRI
       1. 4320 bp

4:  pFC8tacT1T2
       1. 3916 bp

5:  pFC8tacT1T2
    BamHI + NotI
       1. 3306 bp
       2. 589 bp
       3. 21 bp

6:  pFC8tacT1T2
    EcoRI + XbaI
       1. 2574 bp
       2. 818 bp
       3. 524 bp
```

### Results

Ran 0.08 agarose gel in TAE for 45 mins at 120V (shown below). R are
Rachel's plasmids while EH are my own.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC53tac-pfc8tac-Rachel-ethan-digest-labeled.png)

All digests look basically the same and match the expected results for
the simulated digest of pFC8tac. At this point I would say with confidence
that Rachel's pFC53tac tube should be relabeled as pFC8tacT1T2. The double
band produced by the EcoRI digest confirms the presence of the T1T2 terminators.

### pFC primer design

Primer design for sequencing pFC series plasmids. In order to be absolutely
sure about the identity of each plasmid I will sequence each plasmid I am
planning on working with using the primers in the below sections.
This is my first time designing primers for the lab
and so to keep track of them I created a
[spreadsheet](https://docs.google.com/spreadsheets/d/1hC9VYnkETDaF73ewFpsUHhBizdWiu8ck4LPnwME005o/edit?usp=sharing) were details of all future oligos
should be stored.

#### [pFC8tac](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/sequences/pFC8_t7_Amplified.dna)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC8_tac_amp_Map.png)

##### Sequences

```
>pFC8tac_tac_promoter_Primer_1
gcagaacggcacaacagc
>pFC8tac_tac_promoter_Primer_2
gtattaccgcctttgagtgagctg
```

##### PCR program

```
|95°C|95°C               |    |tmf:64.1
|____|_____          72°C|72°C|tmr:66.4
|5min|30s  \ 60.7°C _____|____|45s/kb
|    |      \______/ 0:30|5min|GC 55%
|    |       30s         |    |301bp
```

#### [pFC9](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/sequences/pFC9_t7_Amplified.dna)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC9_t7_Amplified_Map.png)

##### Sequences

```
>pFC9_t7_primer_1
cccgccgcgcttaatg
>pFC9_t7_primer_2
gccccaatgcgagcgg
```

##### PCR program

```
|95°C|95°C               |    |tmf:66.0
|____|_____          72°C|72°C|tmr:69.1
|5min|30s  \ 61.3°C _____|____|45s/kb
|    |      \______/ 0:30|5min|GC 56%
|    |       30s         |    |300bp
```

#### [pFC8](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/sequences/pFC8_tac_amp.dna)

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/pFC8_t7_Amplified_Map.png)

##### Sequences

```
>pFC8_t7_primer_2
gaaggggcagtagcacagtcc
```

##### PCR program

```
|95°C|95°C               |    |tmf:66.0
|____|_____          72°C|72°C|tmr:69.5
|5min|30s  \ 61.3°C _____|____|45s/kb
|    |      \______/ 0:30|5min|GC 55%
|    |       30s         |    |319bp
```

All PCR programs calculated using pydna `pcr` module for Tac polymerase.
Notebook for protocol generation can be found [here](notebooks/7-29-21-pcr-rxns.ipynb) or [pdf version here](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/notebooks/exported_notebooks/7-29-21-pcr-rxns.pdf).

### Insert sequence primers

Since I was ordering primers I also added primers that will amplify all variable regions inserts in case we need to make a larger amount of any
insert.

![](/home/ethan/Documents/github/lab-notes/experiments/VR-inserts/images/Insert_pcr_product_Map.png)

#### Sequences

```
>Variable_region_insert_primer_1
tacgactcactatagggcgaattgg
>Variable_region_insert_primer_2
cctcctcgcctcggtcac
```

### PCR protocol outline

Reagent spreadsheet [available at this link](https://docs.google.com/spreadsheets/d/1C9dQ5NALOPIBd9vnqTwMcuQwFouvtItC6r6D7yj8_8g/edit?usp=sharing).

